docs: add FAQ regarding GOROOT env var (#3122)
operator-framework/operator-sdk: website/content/en/docs/faq.md

	}
```

## I see deepcopy errors and image build fails. How do I fix this?

When you run the ```operator-sdk generate k8s``` command, you might see an error like this

> ``` bash

Reviewer: For proper rendering
````suggestion
```
````

Author: done

Reviewer: Still shows up as `bash`.

Author: fixed now

INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [cache:[v1alpha1], ] 
F0523 01:18:27.122034    5157 deepcopy.go:885] Hit an unsupported type invalid type for invalid type, from github.com/example-inc/memcached-operator/pkg/apis/cache/v1alpha1.Memcached
```

> This is because of the ```GOROOT``` environment variable not being set. More details [here][goroot-github-issue].

Reviewer 2: ```suggestion
This is because of the `GOROOT` environment variable not being set. More details [here][goroot-github-issue].
```

Reviewer: We don't necessarily need to explain what this variable does.

Author: I agree.

> In order to fix this, you simply need to export the ```GOROOT``` environment variable

Reviewer 2: ```suggestion
In order to fix this, you simply need to export the `GOROOT` environment variable
```

Author: I'll add this.

``` bash

> export GOROOT=$(go env GOROOT)

Reviewer: ```suggestion
$ export GOROOT=$(go env GOROOT)
```

Author: done

```

> This will work for the current terminal instance, to make it more permanent, consider adding this line to the ```bashrc``` file.

Reviewer: ```suggestion
This will work for the current environment. To persist this fix, add the above line to your environment's config file, ex. `.bashrc`.
```

Author: done

	}
```

> ## When using the "Getting Started" guide, I see deepcopy errors and image build fails. How do I fix this?

Reviewer 2: ```suggestion
## I see deepcopy errors and image build fails. How do I fix this?
```

> 

Reviewer 2: Not just valid for Getting started. so should not really be so specific. WDYT? 
could we try to re-phase it?

Author: I think you're right, this error is more generic, so we can keep the FAQ generic as well.

Reviewer 2: could we try to shape the following description accordingly as well too?

Author: Oh, sure, let me fix it real quick.
